### PR TITLE
Makefile equiv Powershell script

### PR DESCRIPTION
**Pull request by ashley-cui**
Introduce a powershell script that mirrors Makefile capbilities on
Windows.

Syntax: ./winmake target [options]

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Comment by ashley-cui** [comment]
@baude PTAL

File: pkg/machine/qemu/machine.go [hunk 1]
```diff
@@ -589,11 +588,7 @@ func (v *MachineVM) qemuPid() (int, error) {
 		logrus.Warnf("Reading QEMU pidfile: %v", err)
 		return -1, nil
 	}
-
-	if err := unix.Kill(pid, 0); err != nil {
-		if err == unix.ESRCH {
-			return -1, nil
-		}
+	if err := killIfFound(pid); err != nil {
```
Comment (Luap99) [inline pkg/machine/qemu/machine.go:591]: this is not equivalent, now it will not return -1,nil for ESRCH which I assume is incorrect.

Comment (ashley-cui) [reply pkg/machine/qemu/machine.go:591]: oops, thanks!

**Comment by ashley-cui** [comment]
Weird, this PR isn't updating with my force-pushes. Closing and re-opening to see if it updates then.

**Comment by ashley-cui** [comment]
Opened https://github.com/containers/podman/pull/20031

**Pull request closed without merging**